sphinxcontrib/openapi/openapi30.py

```diff
@@ -246,6 +248,49 @@ def _httpresource(endpoint, method, properties, convert, render_examples,
             yield '{indent}{line}'.format(**locals())
         yield ''
 
+    securities = properties.get('security', [])
```

Comment:
It does not look like a `oneOf` support. Can you please move security schemes support to separate PR?